src/pages/index.js

```diff
@@ -54,9 +55,35 @@ const IndexPage = (props) => {
                         </Box>
                     </Box>
                 </Link>
-            )
+            );
         }
-    })
+    });
+
+    const [thankYou, setThankYou] = React.useState([]);
+
+    useEffect(() => {
+        axios
+            .get(
+                'https://raw.githubusercontent.com/jmMeessen/jenkins-submitter-stats/main/data/honored_contributor.csv',
```

Comment:
```suggestion
                'https://raw.githubusercontent.com/jenkins-infra/jenkins-contributor-stats/main/data/honored_contributor.csv',
```

Reply:
Thank you @krisstern for making the edits..it's looking great imo!